README.md

```diff
@@ -19,6 +19,7 @@ import { Client } from 'ilp-core'
 
 const client = new Client({
```

Comment:
Is the `Client` still the "recommended" way to use this module now? Should `Core` be documented here instead?